composer.json

```diff
@@ -25,7 +27,10 @@
         "ext-mbstring": "*",
         "ext-curl": "*",
         "composer/ca-bundle": "~1.0",
-        "symfony/options-resolver": "~2.7|~3.0"
+        "symfony/options-resolver": "~2.7|~3.0",
+        "php-http/httplug": "~1.1",
+        "php-http/curl-client": "~1.7",
```

Comment:
>Well, installing a client by default became a common practice, even if it kinda destroys one of the original purposes

Does this totally defat the purpose of Composer virtual packages, right? By the way I don't see a really big problem to ask the user to choose one client library and not rely on any one provided out-of-the-box, if the README is clear enough to explain why and how he must proceed. As this is indeed a blocking topic, @dcramer can you please give us your opinion on shipping with a default client like cURL?